packages/st2mistral/Makefile

```diff
@@ -8,7 +8,7 @@ PBR_GITURL := git+https://github.com/openstack-dev/pbr@d19459daa8616dd18fde016c2
 
 # We use special additional requirements in our mistral bundle!
 define INJECT_DEPS
-git+https://github.com/StackStorm/st2mistral.git@$(GITREV)#egg=st2mistral
+git+https://github.com/StackStorm/st2mistral.git@st2-$(MISTRAL_VERSION)#egg=st2mistral
```

Comment:
So here is where `gitrev`: https://github.com/StackStorm/st2-packages/blob/master/rake/build/environment.rb#L85 env is populated for `st2mistral` package.

It already looks good.
So if we need to change the stuff, - change that line.

Let me look at the [`v2.2` change](https://github.com/StackStorm/st2-packages/pull/452)